job/storage/postgres/postgres.go

```diff
@@ -0,0 +1,108 @@
+package postgres
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+
+	_ "github.com/lib/pq"
+
+	"github.com/ajvb/kala/job"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var (
+	table = "jobs"
```

Comment:
Should this be a `const`? Can we give it a more descriptive name as well?

Reply:
yes, though I noticed all other storage implementations define as `var` so I thought I'd try to keep consistent.